src/api_server/swagger/firecracker.yaml

```diff
@@ -664,6 +688,14 @@ definitions:
         description: The amount of milliseconds it takes for the bucket to refill.
         minimum: 0
 
+  MmdsConfig:
```

Comment:
Hmm, what I was trying to say here by alphabetical order was to put this definition between `Metrics` and `NetworkInterface`. Sorry for not being clearer.